gdx/src/com/badlogic/gdx/graphics/g2d/SpriteBatch.java

```diff
@@ -331,25 +358,29 @@ else if (idx == vertices.length) //
 		vertices[idx++] = y1;
 		vertices[idx++] = color;
 		vertices[idx++] = u;
-		vertices[idx++] = v;
+		vertices[idx] = v;
+		idx += extrasStride;

```

Comment:
Oops! Can we even afford the single field lookup on something called so often on so many existing projects?